Xamarin.Forms.Core/MultiBinding.cs

```diff
@@ -13,12 +10,12 @@ public sealed class MultiBinding : BindingBase
 	{
```

Comment:
review this file using split mode. the diff doesn't make sense as most of the code have been changed